### DISCUSSION TITLE
Failed to download cli binaries

### DISCUSSION THREAD

**Discussion by ghost** [Brendonovich/prisma-client-rust#251] [Q&A] [answered]
Hello, I was just following the [installation](https://prisma.brendonovich.dev/getting-started/installation), but wasn't able to download the binary after run `cargo prisma generate`... it just stuck there

<img width="657" alt="Screenshot 2023-02-07 at 3 27 48 PM" src="https://user-images.githubusercontent.com/8101584/217177783-424cde5e-57b4-450d-b139-f6bef52f520c.png">

<img width="243" alt="Screenshot 2023-02-07 at 3 28 21 PM" src="https://user-images.githubusercontent.com/8101584/217177985-c00a6566-ac2e-495c-b3ef-249a71c33899.png">

I can download with the same url from browser.

<img width="524" alt="Screenshot 2023-02-07 at 3 15 05 PM" src="https://user-images.githubusercontent.com/8101584/217178344-a3c31cdc-b438-4342-a8b3-84cbfd1d095d.png">

And also if I put the extracted binary to the folder (which is `users/name/library/caches/prisma/binaries/cli/4.8.0/xxxx`), it would still try to download again.

Any guidance would be appreciated.

**Comment by Brendonovich** [comment, marked as answer]
I've seen this happen before but not this recently, I'm not quite sure what's going on. What version of rust are you using?

**Reply by ghost** [reply]
Hi thanks for reply, it's 1.69.0-nightly

**Reply by Brendonovich** [reply]
Can you try using stable? PCR doesn't officially support nightly.

**Reply by ghost** [reply]
Thanks, it works now after switching back to stable. :)
